cmd/submit.go

```diff
@@ -97,6 +99,28 @@ Your submission can be found online at %s
 	if submission.Iteration == 1 {
 		msg += `
 To get the next exercise, run "exercism fetch" again.
+
+`
+		rand.Seed( time.Now().UTC().UnixNano() )
+		phrases := []string{
+			"For bonus points",
+			"Don't stop now: The fun's just begun",
+			"Some tips to continue",
+		}
+		msg += fmt.Sprintf("## %s", phrases[rand.Intn(len(phrases))])
+		msg += `
+
+Did you get the tests passing and the code clean? If you want to, these are some
+additional things you could try:
+
+* Remove as much duplication as you possibly can.
+* Optimize for readability, even if it means introducing duplication.
+* If you've removed all the duplication, do you have a lot of conditionals? Try
+  replacing the conditionals with polymorphism, if it applies in this language.
```

Comment:
Let's not mention polymorphism. How about suggesting trying to get rid of conditionals without suggesting a tactic?